lib/presentation/shared_widgets/crowdaction_card.dart

```diff
@@ -49,41 +50,53 @@ class CrowdActionCard extends StatelessWidget {
                 ),
               ),
             ),
-            Padding(
-              padding: const EdgeInsets.fromLTRB(15.0, 15.0, 15.0, 30.0),
-              child: Column(
-                children: [
-                  SingleChildScrollView(
-                    scrollDirection: Axis.horizontal,
-                    child: Wrap(
-                      spacing: 12.0,
-                      children: chips,
-                    ),
+            const SizedBox(height: 5.0),
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                SingleChildScrollView(
+                  scrollDirection: Axis.horizontal,
+                  child: Row(
+                    children: [
+                      const SizedBox(width: 15.0),
+                      Wrap(
+                        spacing: 12.0,
+                        children: chips,
+                      ),
+                    ],
                   ),
+                ),
+                const SizedBox(height: 5.0),
+                Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 15.0),
+                  child: Column(
+                    children: [
+                      Text(
+                        title,
+                        maxLines: 3,
+                        overflow: TextOverflow.ellipsis,
+                        style: const TextStyle(
+                            fontSize: 22.0,
+                            fontWeight: FontWeight.bold,
+                            color: kPrimaryColor400),
+                      ),
+                    ],
+                  ),
+                ),
+                if (description != null) ...[
                   const SizedBox(height: 18.0),
                   Text(
```

Comment:
The `description` Text widget is outside the Padding. So, it is sticking to the edges of the card and doesn't look good.

Everything else looks great!

Reply:
Haha yes, my bad. I changed it so that the chips, if scrollable, does not have Padding  forgot the description. I will fix.